environments/dotnet/FissionContext.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+
+namespace Fission.DotNetCore.Api
+{
+    public class FissionContext
+    {
+        public FissionContext(Dictionary<string, object> args, Logger logger)
+        {
+            if (args == null) throw new ArgumentNullException(nameof(args));
+            if (logger == null) throw new ArgumentNullException(nameof(logger));
+            Arguments = args;
+            Logger = logger;
+        }
+
+        public Dictionary<string, object> Arguments { get; private set; }
+        public Logger Logger { get; private set; }
+    }
+
+    public class Logger
```

Comment:
I'll look into it and get back to you. I just needed to know if you already had something so I should go look for a .net port of that. Give me a few days and I'll figure it out.